loomi/react_boilerplate
# Feature/prettier config

**What I did:**
- Updated the dependencies
- Fix ESLint configuration file
- Create Prettier configuration file
- Install ESLint/Prettier plugins and libraries
- Create VSCode settings file
- Add Lint action (to Github)


**How to test:**
1. Install the dependencies, using `yarn` or `npm i`
1. Try to run the project, using `yarn start`or `npm start`
1. Check if the code formats (with the ESLint rules) on file save
